packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -119,6 +120,11 @@ class ExportToCsvCommand extends Command {
                 apiKey: listOfTokens[managementTokenAlias].apiKey,
                 token: listOfTokens[managementTokenAlias].token,
               };
+              const checkManagementTokenValidity = await isManagementTokenValid(stackAPIKey,listOfTokens[managementTokenAlias].token);
+              if(!checkManagementTokenValidity?.valid) {
+                cliux.print(`error: Management token or stack API key is invalid. ${checkManagementTokenValidity?.message||""}`, {color: "red"});
+                process.exit(1);
```

Comment:
@cs-raj Try to avoid exiting the process in the middle of the code. Doing so will prevent you from being able to cover those lines of code when writing test cases.

Reply:
Removed the process.exit